Floofbot/Modules/Utilities.cs

```diff
@@ -256,4 +258,163 @@ public async Task ServerList()
             }, true);
         }
     }
+    [Summary("Conversion commands")]
+    [Discord.Commands.Name("Conversions")]
+    public class Conversions : InteractiveBase
+    {
+        private static readonly Discord.Color EMBED_COLOR = Color.Magenta;
+
+        [Command("convert")]
+        [Alias("conv")]
+        [Summary("Converts units to other units, such as Celcius to Fahrenheit.")]
+
+        public async Task convert(string input)
+        {
+            Regex fahReg = new Regex("\\b(\\d+)(f)\\b", RegexOptions.IgnoreCase);
+            Regex celReg = new Regex("\\b(\\d+)(c)\\b", RegexOptions.IgnoreCase);
+            Regex miReg = new Regex("\\b(\\d+)(mi)\\b", RegexOptions.IgnoreCase);
+            Regex kmReg = new Regex("\\b(\\d+)(km)\\b", RegexOptions.IgnoreCase);
+            Regex kgReg = new Regex("\\b(\\d+)(kg)\\b", RegexOptions.IgnoreCase);
+            Regex lbReg = new Regex("\\b(\\d+)(lbs)\\b", RegexOptions.IgnoreCase);
+
+            if (fahReg.Match(input).Success)
+            {
+                Match m = fahReg.Match(input);
+
+                Group g = m.Groups[1];
```

Comment:
is there a need to use groups? I believe match returns the first result anyway, or am I missing something?

Reply:
Actually, the match returns the whole string, that's why I needed to use the groups

Reply:
You can use positive lookahead instead of groups, eg

`\d+(?=f )`